src/app/use-cases/counter.ts

```diff
@@ -43,19 +43,11 @@ export const selectors = {
 };
 
 export const thunks = {
-  ...(() => {
+  incrementAsync: createAsyncThunk<number, number, AsyncThunkConfig>(`${name}/incrementAsync`, async (amount: number, { extra }) => {
```

Comment:
```diff
- incrementAsync: createAsyncThunk<number, number, AsyncThunkConfig>(`${name}/incrementAsync`, async (amount: number, { extra }) => {
+ incrementAsync: createAsyncThunk<number, number, AsyncThunkConfig>(`${name}/incrementAsync`, async (amount, { extra }) => {
```
Here, we don't need type annotaition for the type of `ammount` it's constrained by the second type argument of `createAsyncThunk`